### ISSUE TITLE
nuke :setup generated project fails with 'AbsolutePath' could not be found

### ISSUE THREAD

**Issue by arialdomartini** [nuke-build/nuke#433]
### Description
This is a duplicate of #403 opened by @gsscoder: sorry if I open a new issue, but #403 has been locked.

### Steps to reproduce

1. Update NUKE Global Tool to 0.24.4

```
dotnet tool update --global Nuke.GlobalTool
```
2. Create an empty .NET Core project

3. Generate a sample script with 

```
nuke :setup
```

4. Run 

```
nuke compile
```

The build fails with

```
NUKE Global Tool version 0.24.4 (Windows,.NETCoreApp,Version=v2.1)
PowerShell Desktop version 5.1.18362.628
Microsoft (R) .NET Core SDK version 3.1.101
Build.cs(31,5): error CS0246: The type or namespace name 'AbsolutePath' could not be found (are you missing a using directive or an assembly reference?) [C:\prg\_kod\MyServez1\build\_build.csproj]
```

My feeling is NUKE GlobalTool is using an old template.

**Comment by arialdomartini** [comment]
I noticed that the missing using statement is:

```
using Nuke.Common.IO;
```

which is in fact not present in [Build.cs](https://github.com/nuke-build/nuke/blob/aa101837848cc062f03970a50c70fdb6469bdf44/source/Nuke.GlobalTool/templates/Build.cs) @ aa10183784

**Comment by matkoch** [comment]
Maybe submit a PR? :)

**Comment by arialdomartini** [comment]
Sure, I will! 
So, you are confirming the missing `using` is a bug, aren't you?

Sorry for asking, I'm not expert with NUKE's internals: we got mad for about 2 hours a remote colleague and I: she wasn't able to replicate the issue I was having, so I was sure that depended on an old template still around in my installation.

Also, #403 seems to confirm that this wasn't a bug.

**Comment by arialdomartini** [comment]
Ok, just published #435. 

It is still a mystery to me why in #403 @matkoch commented

> I just tested the setup procedure calling nuke twice (first execution is a setup, 
> second will build it). Everything works as expected. Could you point me to a repro? 

It seems to suggest that this was *not* a bug, but something depending on the local environment.

Before pushing a PR, I firstly opened an issue as suggested by your Contribution Guideline (at Don't Push Your Pull Requests"). I apologize if I have misunderstood the meaning of 

> before you dive into the editor, open a discussion, outline your problem and solution 
> and offer to do the work

Regarding #435, I wanted to run `nuke test` before and after my change, but found that `nuke test` fails in master, with the following error:

```
The report file pattern 'C:\prg\_kod\nuke\output\test-results\*.xml' is invalid. No matching files found.
No report files specified.
Process 'dotnet.exe' exited with code 1.
   > C:\prg\_kod\nuke\.tmp\dotnet-win\dotnet.exe C:\Users\arialdo.martini\.nuget\packages\reportgenerator\4.2.16\tools\netcoreapp2.1\ReportGenerator.dll -reports:C:\prg\_kod\nuke\output\test-results\*.xml -targetdir:C:\prg\_kod\nuke\output\coverage-report -reporttypes:HtmlInline
   @ C:\prg\_kod\nuke

   at Nuke.Common.Tooling.ProcessExtensions.AssertZeroExitCode(IProcess process) in C:\prg\_kod\nuke\source\Nuke.Common\Tooling\ProcessExtensions.cs:line 37
   at Nuke.Common.Tools.ReportGenerator.ReportGeneratorTasks.ReportGenerator(ReportGeneratorSettings toolSettings) in C:\prg\_kod\nuke\source\Nuke.Common\Tools\ReportGenerator\ReportGenerator.Generated.cs:line 69
   at Nuke.Common.Tools.ReportGenerator.ReportGeneratorTasks.ReportGenerator(Configure`1 configurator) in C:\prg\_kod\nuke\source\Nuke.Common\Tools\ReportGenerator\ReportGenerator.Generated.cs:line 93
   at Build.<get_Coverage>b__65_1() in C:\prg\_kod\nuke\build\Build.cs:line 218
   at Nuke.Common.Execution.BuildExecutor.<>c.<Execute>b__4_0(Action x) in C:\prg\_kod\nuke\source\Nuke.Common\Execution\BuildExecutor.cs:line 109
   at Nuke.Common.Utilities.Collections.EnumerableExtensions.ForEach[T](IEnumerable`1 enumerable, Action`1 action) in C:\prg\_kod\nuke\source\Nuke.Common\Utilities\Collections\Enumerable.ForEach.cs:line 17
   at Nuke.Common.Execution.BuildExecutor.Execute(NukeBuild build, ExecutableTarget target, IReadOnlyCollection`1 previouslyExecutedTargets, Boolean failureMode) in C:\prg\_kod\nuke\source\Nuke.Common\Execution\BuildExecutor.cs:line 109

Repeating warnings and errors:
Assertion failed: Could not complete checking build configurations within 500 milliseconds.
Process 'dotnet.exe' exited with code 1.
   > C:\prg\_kod\nuke\.tmp\dotnet-win\dotnet.exe C:\Users\arialdo.martini\.nuget\packages\reportgenerator\4.2.16\tools\netcoreapp2.1\ReportGenerator.dll -reports:C:\prg\_kod\nuke\output\test-results\*.xml -targetdir:C:\prg\_kod\nuke\output\coverage-report -reporttypes:HtmlInline
   @ C:\prg\_kod\nuke
```

I'm investigating on this and possibly opening a new issue.
